tests/test_create_or_update_jobs.py

```diff
@@ -210,3 +211,9 @@ def make_job_request(action="generate_cohort", **kwargs):
     for key, value in kwargs.items():
         setattr(job_request, key, value)
     return job_request
+
+
+def create_jobs_with_project_file(job_request, project_file):
+    with mock.patch("jobrunner.create_or_update_jobs.get_project_file") as f:
```

Comment:
Is mocking unavoidable here?

Reply:
It might be, but I'm not quite sure. The reason I used it here was that it was the easiest way of getting the existing tests working without doing some hard thinking and rewriting them, which I didn't fancy doing right then.